strings/credits.yml

```diff
@@ -5,1242 +5,1420 @@ Nothing:
   Potion Shop: "the witch buying drugs"
   Master Sword Pedestal: "and the couch cash"
   Sick Kid: "the piggy-bank kid"
   Flute Spot: "destitute boy has lunch again"
   Link's Uncle: "your uncle likes cash"
+  Blacksmith: "the rupeesmiths"
 PendantOfCourage:
-  King Zora: "Nothing, so stupid"
-  Potion Shop: "mushrooms go poof"
-  Master Sword Pedestal: "and nothing"
-  Sick Kid: "nothing-having kid"
-  Flute Spot: "empty boy does nothing again"
-  Link's Uncle: "your uncle does nothing"
+  King Zora: "valor for sale"
+  Potion Shop: "mushrooms prove courage"
+  Master Sword Pedestal: "the wind necklace"
+  Sick Kid: "courage-judging kid"
+  Flute Spot: "courage boy roars again"
+  Link's Uncle: "your uncle is brave"
+  Blacksmith: "the medalsmiths"
 PendantOfWisdom:
-  King Zora: "Nothing, so stupid"
-  Potion Shop: "mushrooms go poof"
-  Master Sword Pedestal: "and nothing"
-  Sick Kid: "nothing-having kid"
-  Flute Spot: "empty boy does nothing again"
-  Link's Uncle: "your uncle does nothing"
+  King Zora: "brains for sale"
+  Potion Shop: "mushrooms prove wisdom"
+  Master Sword Pedestal: "the love necklace"
+  Sick Kid: "wisdom-judging kid"
+  Flute Spot: "wisdom boy scares crows again"
+  Link's Uncle: "your uncle is wise"
+  Blacksmith: "the medalsmiths"
 PendantOfPower:
-  King Zora: "Nothing, so stupid"
-  Potion Shop: "mushrooms go poof"
-  Master Sword Pedestal: "and nothing"
-  Sick Kid: "nothing-having kid"
-  Flute Spot: "empty boy does nothing again"
-  Link's Uncle: "your uncle does nothing"
+  King Zora: "potency for sale"
+  Potion Shop: "mushrooms prove power"
+  Master Sword Pedestal: "the fire necklace"
+  Sick Kid: "power-judging kid"
+  Flute Spot: "power boy chops trees again"
+  Link's Uncle: "your uncle is strong"
+  Blacksmith: "the medalsmiths"
 BowAndArrows:
   King Zora: "point stick for sale"
   Potion Shop: "witch and robin hood"
   Master Sword Pedestal: "the stick and twine"
   Sick Kid: "arrow-slinging kid"
   Flute Spot: "archer boy shoots again"
   Link's Uncle: "your uncle, robin hood"
+  Blacksmith: "the bowyers"
 BowAndSilverArrows:
   King Zora: "you got lucky"
   Potion Shop: "witch and robin hood"
   Master Sword Pedestal: "the stick and shine"
   Sick Kid: "arrow-slinging kid"
   Flute Spot: "archer boy shoots again"
   Link's Uncle: "your uncle, robin hood"
+  Blacksmith: "the electroplating bowyers"
 BottleWithBee:
   King Zora: "mad friend for sale"
   Potion Shop: "insects for trade"
   Master Sword Pedestal: "and the mad friend"
   Sick Kid: "the bug-caught kid"
   Flute Spot: "bottle boy has mad bee again"
   Link's Uncle: "your uncle likes arthropods"
+  Blacksmith: "the beekeepers"
 BottleWithFairy:
   King Zora: "friend for sale"
   Potion Shop: "shrooms for friends"
   Master Sword Pedestal: "and the fairy friend"
   Sick Kid: "fairy-catching kid"
   Flute Spot: "bottle boy has friend again"
   Link's Uncle: "your uncle has a friend"
+  Blacksmith: "don't ask"
```

Comment:
lol.

Reply:
Glad you like my approach to keeping it G-rated there.  ;)